flake.nix

```diff
@@ -181,7 +195,13 @@
 
         desktop = makeApp {
           app = "desktop";
-          preBuildCommands = "export npm_config_nodedir=${electron.headers}; pnpm postinstall";
+          # pnpm throws an error at the end of `pnpm postinstall`, but it doesn't seem to matter:
+          # ENOENT: no such file or directory, lstat
+          # '/build/source/apps/desktop/node_modules/better-sqlite3/build/node_gyp_bins'
```

Comment:
I did notice something interesting about it, this one is triggered by the second rebuild, done for `edit-docs` and not for the `desktop`. This is probably also why it doesn't break anything.

I'm not sure why `electron-rebuild` is touching the `desktop` again, even though it's rebuilding `edit-docs` with its own dependency.